### PR TITLE
Populating functions

### PR DESCRIPTION
**Pull request by aryansri-19**
action added to director's corner component

### CODE REVIEW

File: server/schema/clubs.schema.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   integer,
   pgTable,
   smallint,
-  smallserial,
+  serial,
```
Comment (GetPsyched) [inline server/schema/clubs.schema.ts:8]: Why?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: server/schema/clubs.schema.ts (hunk 1 above)
Comment (aryansri-19) [reply server/schema/clubs.schema.ts:8]: because smallserial is not recognized by typescript as a default auto-incrementing type, and thus, it always asks to include id also in the data which should not be the case.

Comment (GetPsyched) [reply server/schema/clubs.schema.ts:8]: This has been fixed in [`drizzle-orm v0.30.5`](https://github.com/drizzle-team/drizzle-orm/releases/tag/0.30.5). Drizzle was updated in master to the latest version, just rebase this branch and revert back to using `smallserial`

Comment (aryansri-19) [reply server/schema/clubs.schema.ts:8]: oh, alright

File: server/schema/deans.schema.ts [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
-  text,
-  varchar,
-} from 'drizzle-orm/pg-core';
+import { integer, pgTable, serial, text, varchar } from 'drizzle-orm/pg-core';
 
 import { faculty } from '.';
 
 export const deans = pgTable('deans', {
-  id: smallserial('id').primaryKey(),
+  id: serial('id').primaryKey(),
```
Comment (GetPsyched) [inline server/schema/deans.schema.ts:7]: Why was this changed? Please don't change DB datatypes without prior discussion.

File: server/schema/departments.schema.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {
 export const departments = pgTable('departments', {
   id: smallserial('id').primaryKey(),
   name: varchar('name', { length: 64 }).notNull(),
-  alias: char('alias', { length: 2 }).notNull(),
+  alias: char('alias', { length: 4 }).notNull(),
```
Comment (GetPsyched) [inline server/schema/departments.schema.ts:16]: We will change this later if necessary. Aliases should be only 2 chars.

File: server/schema/hod.schema.ts [hunk 4]
```diff
@@ -21,7 +21,7 @@ export const hod = pgTable('hod', {
     .notNull(),
   message: text('message'),
   isActive: boolean('is_active').default(true).notNull(),
-  createdOn: date('created_on').defaultNow().notNull(),
+  createdOn: timestamp('created_on').defaultNow().notNull(),
```
Comment (GetPsyched) [inline server/schema/hod.schema.ts:24]: This will remain a date, we don't know or care what time of the day did a person become an HOD

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: server/schema/departments.schema.ts (hunk 3 above)
Comment (aryansri-19) [reply server/schema/departments.schema.ts:16]: For some, like Chemistry and Electronics, they were requiring 3-4 letters, thus I changed it. Will revert back

File: server/schema/deans.schema.ts (hunk 2 above)
Comment (aryansri-19) [reply server/schema/deans.schema.ts:7]: I did rebase it though, i don't know how it didn't get changed back.

File: server/schema/hod.schema.ts (hunk 4 above)
Comment (GetPsyched) [reply server/schema/hod.schema.ts:24]: Maybe try this:
```suggestion
  createdOn: date('created_on', { mode: 'date' }).defaultNow().notNull(),
```

File: tsconfig.json [hunk 5]
```diff
@@ -6,8 +6,7 @@
     "isolatedModules": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "target": "es6",
-
+    "target": "es2017",
```
Comment (GetPsyched) [inline tsconfig.json:9]: Reason?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 5 above)
Comment (aryansri-19) [reply tsconfig.json:9]: Oh, it was to allow root level await functionality, that I used for testing, will revert it.

File: server/schema/hod.schema.ts (hunk 4 above)
Comment (GetPsyched) [reply server/schema/hod.schema.ts]: @aryansri-19 did this work?

Comment (aryansri-19) [reply server/schema/hod.schema.ts]: Yeah, changed this back in the latest push.

**Comment by GetPsyched** [comment]
Thanks @aryansri-19. Will be closing this PR as we can just use its contents without merging.
